packages/client/src/utils/index.ts

```diff
@@ -168,3 +167,57 @@ export const getAnswerLabel = (answer: ClientSimpleType) => {
   }
   return answer;
 };
+
+/**
+ *
+ * Validates if the corresponding slug has a transformed IMTR file
+ * `apiTopics` contains all files in `/public/imtr/transformed/`
+ *
+ * @param {Topic} topic
+ */
+export const hasIMTR = (topic?: Topic) => {
+  return !!(
+    topic?.slug && apiTopics.flat().find((api) => api.slug === topic.slug)
+  );
+};
+
+/**
+ *
+ * Returns true if an IMTR file is found AND the topic type if equal to `PERMIT_CHECK`
+ * This only returns configured checks and ignores OLO flow permit-checks
+ *
+ * @param {Topic} topic
+ */
+export const isConfiguredPermitCheck = (topic: Topic) => {
+  return topic.type === TopicType.PERMIT_CHECK && hasIMTR(topic);
+};
+
+/**
+ *
+ * Returns true if topic type if equal to `PERMIT_CHECK`
+ *
+ * @param {Topic} topic
+ */
+export const isPermitCheck = (topic?: Topic) => {
+  return topic?.type === TopicType.PERMIT_CHECK;
+};
+
+/**
+ *
+ * Returns true if topic type if equal to `PERMIT_FORM`
+ *
+ * @param {Topic} topic
+ */
+export const isPermitForm = (topic?: Topic) => {
```

Comment:
Maybe you can create a Topic-class that has an attribute isPermitForm that's async / cached. I'll add it to the dev-meeting topics.

Reply:
😄